script.js

```diff
@@ -1,9 +1,21 @@
+// list of all todo items
+
 allTasks = []; // list of all todo items
 completed = []; // list of all items that are currently marked as completed
 uncompleted = []; // list of all items that still need to be completed
 
 loadJSON(`./example.json`); // We should only allow tasks to be added when the loadJSON is finished
 
+// Event listener for day dropdown change
+document.querySelectorAll(".dayDropdown").forEach((dropdown) => {
+  dropdown.addEventListener("toggle", function () {
+    if (this.open) {
+      const selectedDay = this.id.replace("Dropdown", "");
```

Comment:
TLDR: Use [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) instead of ID.

This is **extremely fragile** as its dependent on the developer to name the class id {selectedDay}Dropdown, to make sure that all the tasks are also named in this same way (hence having to strictly make all the strings lowercase later) and make sure that id not used for anything else. 

This would be a good place to use custom components but seeing as that would take too long I would at least use [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) I'm still not entirely sure if theyre appropriate here but it's less dangerous. 

Long term the dropdowns and day options for the tasks should be generated by a list of enums and if there is data that doesnt match this list of enums it should be ignored but I will create a separate issue for that. 

Update: I have created #7  for to create the enum.